packages/GA4Client/src/common.js

```diff
@@ -30,6 +30,10 @@ Common.prototype.mergeObjects = function () {
     return resObj;
 };
 
+Common.prototype.isMpV2 = function (globalMpObject) {
+    return globalMpObject.getVersion().split('.')[0] === '2';
+};
+
```

Comment:
This might be a nitpick but I think this function is too tightly coupled with the SDK. I personally would make it so the function should only expect a version number in the format of `X.XX.X` and then check if that string starts with 2.

So it would become:

```
Common.prototype.isV2 = function (versionString) {
    return versionString.split('.')[0] === '2';
};
```

and to use it:

```
common.isV2(window.mParticle.getVersion())
```